src/Resources/app/storefront/src/js/filter-slider-range.plugin.js

```diff
@@ -318,4 +315,18 @@ export default class FilterSliderRange extends FilterBasePlugin {
       this.slider.noUiSlider.set([null, this._inputMax.value]);
     }
   }
+
+  refreshDisabledState(filter) {
+    const properties = filter['price'];
```

Comment:
What happens for other range-slider filters, if you always explicitly select the `price`?